hack/prefetch-images.sh

```diff
@@ -0,0 +1,57 @@
+set -ex pipefail
+
+source hack/common.sh
+source cluster-up/cluster/$KUBEVIRT_PROVIDER/provider.sh
+source hack/config.sh
+
+function prefetch-images::find_node_names() {
+    if [[ $KUBEVIRT_PROVIDER == "external" ]] || [[ $KUBEVIRT_PROVIDER =~ kind.* ]] || [[ $KUBEVIRT_PROVIDER == "local" ]]; then
+        echo "" # in case of external provider / kind we have no control over the nodes
+    else
+        local nodes=()
+        nodes+=($(_kubectl get nodes -o name | sed "s#node/##g"))
+        echo "${nodes[@]}"
+    fi
+}
+
+# Given a list of images, find nodes, SSH into each node and execute a command to pull image
+function prefetch-images::pull_on_nodes() {
+    local -r containers_to_pull=$@
+    local -r nodes=$(prefetch-images::find_node_names)
+    # only internal providers are supported (we have control over the nodes), and there we know it's docker
+    local -r pull_command="docker"
```

Comment:
nit: ATM I don't see a reason for this local variable to exist, as podman has gone away. Did you keep this because we are going to extend this logic soon to include podman again? If not, I'd suggest inlining this.

Reply:
you are right, at this point time it can be inlined, 
and when the new logic comes in, it will be easy to extend